Specs/Utilities/DOMReady.js

```diff
@@ -6,30 +6,96 @@ provides: ~
 ...
 */
 
-/* todo
-document.addListener = function(type, fn){
-	if (this.addEventListener) this.addEventListener(type, fn, false);
-	else this.attachEvent('on' + type, fn);
-	return this;
-};
-
-document.removeListener = function(type, fn){
-	if (this.removeEventListener) this.removeEventListener(type, fn, false);
-	else this.detachEvent('on' + type, fn);
-	return this;
-};
-
-
-window.fireEvent =
-document.fireEvent = function(type){
-	if (type == 'domready')
-	for (var i = 0; i < domreadyCallbacks.length; ++i){
+describe("DOMReady", function(){
+
+	var win, frame, cb, ready;
```

Comment:
not a fan of abbreviated variable names. code should be readable; let the minimizer worry about brevity. So, `callback` here.